concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-11	11	11	ARM	1673	ARM
+Lundgren-2020-275-12	12	12	ARMADILLO	1885	ARMADILLO
+Lundgren-2020-275-13	13	13	ARRIVE (INTR.)	1387	ARRIVE
+Lundgren-2020-275-14	14	14	ARROW	977	ARROW
+Lundgren-2020-275-15	15	15	ASHES	646	ASH
+Lundgren-2020-275-16	16	16	ATTACH		
+Lundgren-2020-275-17	17	17	AUNT	1272	AUNT
+Lundgren-2020-275-18	18	18	AXE	677	AXE
+Lundgren-2020-275-19	19	19	BACK		
+Lundgren-2020-275-20	20	20	BAD	1292	BAD
+Lundgren-2020-275-21	21	21	BAT	1793	BAT
+Lundgren-2020-275-22	22	22	BATHE (INTR.)	138	BATHE
+Lundgren-2020-275-23	23	23	BE BORN (INTR.)	1090	BE BORN
+Lundgren-2020-275-24	24	24	BE DRY	1398	DRY
+Lundgren-2020-275-25	25	25	BE FAT		
+Lundgren-2020-275-26	26	26	BE ODOROUS		
+Lundgren-2020-275-27	27	27	BE SCARED	3033	SCARED
+Lundgren-2020-275-28	28	28	BEARD	678	BEARD
+Lundgren-2020-275-29	29	29	BEAT	1665	BEAT
+Lundgren-2020-275-30	30	30	BED	1663	BED
+Lundgren-2020-275-31	31	31	BELLY BUTTON	1838	NAVEL
+Lundgren-2020-275-32	32	32	BENT	297	CROOKED
+Lundgren-2020-275-33	33	33	BIG	1202	BIG
+Lundgren-2020-275-34	34	34	BIRD	937	BIRD
+Lundgren-2020-275-35	35	35	BITTER	887	BITTER
+Lundgren-2020-275-36	36	36	BLACK	163	BLACK
+Lundgren-2020-275-37	37	37	BLOOD	946	BLOOD
+Lundgren-2020-275-38	38	38	BOIL (TR.)	2492	BOIL (SOMETHING)
+Lundgren-2020-275-39	39	39	BONE	1394	BONE
+Lundgren-2020-275-40	40	40	BORROW	1823	BORROW
+Lundgren-2020-275-41	41	41	BRAIN	1583	BRAIN
+Lundgren-2020-275-42	42	42	BRANCH	1531	BRANCH
+Lundgren-2020-275-43	43	43	BREAST	1402	BREAST
+Lundgren-2020-275-44	44	44	BRING (TR.)	692	BRING
+Lundgren-2020-275-45	45	45	BROTHER (F. EGO)	560	BROTHER (OF WOMAN)
+Lundgren-2020-275-46	46	46	BURN	2102	BURN
+Lundgren-2020-275-47	47	47	BURST		
+Lundgren-2020-275-48	48	48	BURY (TR.)	1719	BURY
+Lundgren-2020-275-49	49	49	BUTTERFLY	1791	BUTTERFLY
+Lundgren-2020-275-50	50	50	CAIMAN	2438	CAIMAN
+Lundgren-2020-275-51	51	51	CALL (TR.)	1084	CALL
+Lundgren-2020-275-52	52	52	CANDELA~FIRE		
+Lundgren-2020-275-53	53	53	CANOE	1970	CANOE
+Lundgren-2020-275-54	54	54	CATFISH	3777	CATFISH
+Lundgren-2020-275-55	55	55	CHACRA	212	FIELD
+Lundgren-2020-275-56	56	56	CHAMELEON	1555	CHAMELEON
+Lundgren-2020-275-57	57	57	CHEST	1592	CHEST
+Lundgren-2020-275-58	58	58	CICADA	2353	CICADA
+Lundgren-2020-275-59	59	59	CLAY	1399	CLAY
+Lundgren-2020-275-60	60	60	CLOUD~FOG	3275	CLOUD OR FOG
+Lundgren-2020-275-61	61	61	COCKROACH	1575	COCKROACH
+Lundgren-2020-275-62	62	62	COLLARED PECCARY	435	COLLARED PECCARY
+Lundgren-2020-275-63	63	63	COME (INTR.)	1446	COME
+Lundgren-2020-275-64	64	64	COMPANION	1325	FRIEND
+Lundgren-2020-275-65	65	65	COOK	1100	COOK (SOMETHING)
+Lundgren-2020-275-66	66	66	COPULATE	549	COPULATE
+Lundgren-2020-275-67	67	67	CORN	506	MAIZE
+Lundgren-2020-275-68	68	68	COTTON	1850	COTTON
+Lundgren-2020-275-69	69	69	COVER (TR.)	761	COVER
+Lundgren-2020-275-70	70	70	CROSS	1367	CROSS
+Lundgren-2020-275-71	71	71	CRY (INTR.)	1839	CRY
+Lundgren-2020-275-72	72	72	CULTIVATE (TR.)	1541	CULTIVATE
+Lundgren-2020-275-73	73	73	CURE	499	CURE
```

Comment:
verb or noun?

Reply:
dito, what I mentioned above. Important catch, @Schweikhard

Reply:
This is a difficult one. In Kokama this lexeme can act both as a noun and as a verb. In Omagua, however, this lexeme is translated as a noun (source: A Linguistic Analysis of Old Omagua Ecclesiastical Texts by Michael and O'Hagan (2016) (file:///C:/Users/krist/Desktop/A_Linguistic_Analysis_of_Old_Omagua_Eccl.pdf). There is more specific concept for CURE (NOUN) in the Concepticon and there are also a number of unmapped substantive 'cure' instances in other lists. I can add a new concept set then to refer to a medicine or a treatment as a whole (i.e. a hypernym to MEDICINE)